### PR TITLE
Ffmpeg docker test

### PR DESCRIPTION
**Pull request by dxahtepb**
The simpliest (and still the best) way to dockerize required tools is just to download a well-composed docker image (like [jrottenberg/ffmpeg:4.0-ubuntu](https://hub.docker.com/r/jrottenberg/ffmpeg/builds/bw4dqxsfqcjczypuixbmuqr/)). This image provides all neccessary tools and it's only 200Mb in size.

Of course, one could build his/her own docker image according to [ffmpeg_compile_guide](https://trac.ffmpeg.org/wiki/CompilationGuide/Ubuntu), if any additional third-party libraries for ffmpeg are needed (or more tools). There is no need in building libraries from sources, as long as ubuntu ppa contains all neccessary versions. Resulting docker image turned out to be too fat (~700Mb), maybe because of statically linked libraries.

### CODE REVIEW

File: cwl-scripts/ffmpeg-docker-test/ffmpeg_trim.cwl [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+inputs:
+  input_file_link: string
+  trim_seconds: int
+  out_file_name: string
+
+outputs:
+  out_video:
+    type: File
+    outputBinding:
+      glob: "$(inputs.out_file_name).mp4"
```
Comment (andreyrikunov) [inline cwl-scripts/ffmpeg-docker-test/ffmpeg_trim.cwl:33]: how to get the output file out of container?

Comment (dxahtepb) [reply cwl-scripts/ffmpeg-docker-test/ffmpeg_trim.cwl:33]: Cwl will do this job for us automatically.

Comment (dxahtepb) [reply cwl-scripts/ffmpeg-docker-test/ffmpeg_trim.cwl:33]: [https://www.commonwl.org/user_guide/07-containers/index.html](https://www.commonwl.org/user_guide/07-containers/index.html) It connects docker's workdir with cwl tmp dir (it's shown at example run in guide), so outputs are available outside the container. And files specified in outputs array are copied to cwl's workdir from tmp.

**Review by bvc3at — APPROVED** [review]

**Pull request merged**